### PR TITLE
[DO NOT MERGE] Update known-issues.md

### PR DESCRIPTION
**Pull request by blackdwarf**
Add a better blurb about virtual networking. 

skip ci please

### CODE REVIEW

File: Documentation/known-issues.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ brew install openssl
 brew link --force openssl
 ```
 
+Homebrew may also show the following warning:
+
+> Apple has deprecated use of OpenSSL in favor of its own TLS and crypto libraries
+
+This warning is meant for the software that uses OpenSSL (in this case, .NET Core) and not for the end-user that is installing said software. Homebrew installation doesn't touch either the existing Apple crypto libraries or existing OpenSSL 0.9.8 version, so there is no impact on any software that uses either one of those crypto solutions and is already installed.
+
 MacPorts doesn't have the concept of linking, so it is reccomended that you uninstall 0.9.8 version of OpenSSL using the following command:
```
Comment (bartonjs) [inline Documentation/known-issues.md]: Existing typo in "recommended"

**Comment by bartonjs** [comment]
The OpenSSL part LGTM.  Thanks, Z.  (https://github.com/dotnet/corefx/issues/9171#issuecomment-225690458 also has a bit about creating standalone OS X published applications, if you wanted to merge it in here.  Or any of my other words that you want to borrow)

**Comment by blackdwarf** [comment]
Thanks @bartonjs. Merging this now.

**Pull request merged**